src/main/java/uk/gov/hmcts/ccd/domain/service/getevents/AuthorisedGetEventsOperation.java

```diff
@@ -51,6 +51,11 @@ public AuthorisedGetEventsOperation(@Qualifier("classified") GetEventsOperation
         return secureEvents(caseTypeId, auditEvents.get(0).getCaseDataId(),  auditEvents);
     }
 
+    @Override
+    public List<AuditEvent> getEvents(String caseReference) {
```

Comment:
Kiran can you review `AuditEvent` and make sure it's not missing any important event property and if all the properties we are returning are relevant?
Let's give this a thought as as we found out the v1 endpoint was not written by us. Perhaps we can take this chance to improve the json response that is returned

Reply:
furthermore please test this with a citizen in addition to caseworker. The Service that has requested this endpoint will be accessing it with a `citizen` role so we should make sure it's working